tests/serverpod_test_server/test_integration/websockets/method_websockets/method_streams/revoked_authentication_stream_test.dart

```diff
@@ -0,0 +1,255 @@
+    late String authToken;
+    setUp(() async {
+      authToken = const Uuid().v4();
+      authKeyManager.put(authToken);
+    });
+    group('connected to an authenticated streaming method', () {
+      late Completer<dynamic> streamClosedCompleter;
+      late Completer<int> valueReceivedCompleter;
+      late StreamController<int> inStream;
+
+      setUp(() async {
+        streamClosedCompleter = Completer<dynamic>();
+        inStream = StreamController<int>();
+        Stream<int> outStream;
+        {
+          outStream = client.authenticatedMethodStreaming
+              .intEchoStream(inStream.stream);
+          valueReceivedCompleter = Completer<int>();
+          outStream.listen((event) {
+            if (valueReceivedCompleter.isCompleted) {
+              return;
+            }
+            valueReceivedCompleter.complete(event);
+          }, onError: (e) {
+            streamClosedCompleter.complete(e);
+          });
+
+          inStream.add(1);
+          // Validate that the stream works
+          await valueReceivedCompleter.future;
+          assert(valueReceivedCompleter.isCompleted);
+        }
+      });
+
+      test(
+          'when the authenticated user is revoked then the stream is closed with error.',
+          () async {
+        await expectLater(
+          session.messages.revokedAuthentication(
+            authenticatedUserId,
+            RevokedAuthenticationUser(),
+          ),
+          completion(true),
+        );
+
+        await expectLater(streamClosedCompleter.future, completes);
+        var exception = await streamClosedCompleter.future;
+        expect(exception, isA<ServerpodClientException>());
+      });
+
+      test(
+          'when the authentication id is revoked then the stream is closed with error.',
+          () async {
+        await expectLater(
+          session.messages.revokedAuthentication(
+            authenticatedUserId,
+            RevokedAuthenticationAuthId(authId: 'token-$tokenCounter'),
+          ),
+          completion(true),
+        );
+
+        await expectLater(streamClosedCompleter.future, completes);
+        var exception = await streamClosedCompleter.future;
+        expect(exception, isA<ServerpodClientException>());
+      });
+
+      test(
+          'when an unrelated authentication id is revoked then the stream can still be used.',
+          () async {
+        await expectLater(
+          session.messages.revokedAuthentication(
+            authenticatedUserId,
+            RevokedAuthenticationAuthId(authId: 'token-${tokenCounter + 100}'),
+          ),
+          completion(true),
+        );
+
+        valueReceivedCompleter = Completer<int>();
+        inStream.add(2);
+        await expectLater(
+          valueReceivedCompleter.future,
+          completion(2),
+        );
+      });
+
+      test(
+          'when the required scope for endpoint is revoked the stream is closed with error.',
```

Comment:
Nit:  required scope for **an** endpoint is revoked **then** the stream is closed with **an** error.

Reply:
Validate the other test descriptions too.